packages/components/src/List/List.component.js

```diff
@@ -54,12 +54,18 @@ function DisplayModeComponent({ id, useContent, displayMode, list, virtualized }
 		);
 	}
 	if (virtualized) {
+		const classnames = classNames(
+			'tc-list-display',
+			'tc-list-display-virtualized',
```

Comment:
I have added tc-list-display-virtualized to avoid to override overflow-y: hidden; on tc-list-display during the migration